src/export/export_cluster_snapshot_s3_function.py

```diff
@@ -0,0 +1,52 @@
+import os
+import time
+
```

Comment:
Remove whitespaces between imports

Reply:
@stationeros So the blanks between imports are added by PyCharm as the import layout when you click optimize imports cmd